### PR TITLE
Add definition for "preamble"

### PR DESCRIPTION
**Pull request by sackmanna**
added a definition for the term "preamble"

## Author: 

- 

## Language: 

- 

## Terms defined:

- 
- 
- 

## Editor

Assign the PR based on the language to the following person:

| Language   | Person                       |
|:----------:|:----------------------------:|
| Afrikaans  | @jsteyn, @elletjies          |
| Arabic     | @BatoolMM                    |
| English    | @baileythegreen, @zkamvar    |
| French     | @fmichonneau                 |
| Japanese   | @tomkellygenetics, @masamiy  |
| Portuguese | @beatrizmilz                 |
| Spanish    | @ian-flores                  |

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -4426,6 +4426,15 @@
...
       A naming style that separates the parts of a name with underscores, as in `first_second_third`.
 
+- slug: preamble
+  ref:
+    - LaTeX
+  en:
+    term: "preamble"
+    def: >
+      A series of commands prior to the \begin{document} command that define the type of document based on attributes and parameters. 
+      This is also the section of the document where packages are added using the command \usepackage{} to enable additional functionalities.
```
Comment (baileythegreen) [inline glossary.yml:4436]:
```suggestion
      A series of commands, either placed in the main document, or kept in a separate document,
      that are included prior to the \begin{document} command. The preamble defines the type of
      the document, along with other formatting attributes and parameters. This is also the section
      of the document where packages are added using the command \usepackage{} to enable
      additional functionalities, and where custom commands can be defined. 
```

**Review by baileythegreen — CHANGES_REQUESTED** [review]
I've expanded the definition because it seemed limited to how a small project would be laid out. I use LaTeX for some very large projects, and I never include the preamble in the main document itself. In my current one, I actually have a directory of files for specifying things like new commands, colours, the title page, and then the preamble.tex file is just a bunch of \input{} statements.

I am checking to see which of the other editors use LaTeX to request a review.

**Comment by baileythegreen** [comment]
Note: In searching to see if the term `preamble` is used in languages other than LaTeX, I am seeing many references to `boilerplate code`. This is essentially the same concept, but I want to **strongly suggest** that we avoid using this term in the definition as it is jargon-y and may not be well-understood by non-native speakers. I have no idea how well it might translate.

**Comment by zkamvar** [comment]
> I am checking to see which of the other editors use LaTeX to request a review.

@baileythegreen Your changes look good to me. I have also used preambles similarly.

**Comment by ian-flores** [comment]
@baileythegreen I don't want to merge before getting your okay because you were running point on this PR. With the merging of your suggestion can we know merge this PR into main?

**Review by baileythegreen — APPROVED** [review]

**Comment by baileythegreen** [comment]
@ian-flores Yes, I have just been trying to not be really heavy-handed about my suggestions and automatically commit them.

This was more than a reformatting, by the way. There were pretty significant changes to the definition, but Zhian already signed off on that.

**Comment by elletjies** [comment]
@all-contributors please add @sackmanna  for translation

**Pull request merged**